examples/next-graphql/components/helpers/ComponentResolver.tsx

```diff
@@ -55,15 +56,14 @@ const calculateComponent = function (type: string, paragraph: ParagraphFragmentT
     return <ParagraphViewReference paragraph={paragraph as FragmentOf<typeof ParagraphViewReferenceFragment>} />;
   }
 
-  return <pre>{JSON.stringify(paragraph, null, 2)}</pre>;
+  return <>{JSON.stringify(paragraph, null, 2)}</>;
 }
 
-export const resolve = ({data = [], environment = 'preview'}: ResolveProps): ComponentType => {
+export const resolve = ({data = []}: ResolveProps): ComponentType => {
   if (!data) {
     return []
   }
 
-
   const paragraphUnionFragment = readFragment(ParagraphUnionFragment, data); 
   const components: Array<JSX.Element> = [];
```

Comment:
It can be a simple map
```tsx
const components = paragraphUnionFragment.map((paragraph) => {
    const type = paragraph.__typename;

    if (!type) {
      return <></>;
    }

    return calculateComponent(type, paragraph);
  });
  ```